content/800-data-platform/100-accelerate/100-what-is-accelerate.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'What is Accelerate'
+metaTitle: 'What is Accelerate'
+metaDescription: 'Learn about Accelerate, a global cache and serverless connection pool that makes your database queries faster.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+[Accelerate](https://www.prisma.io/data-platform/accelerate) is a global database cache available in 300 locations that you can use to achieve up to 1000x faster database queries.
+
+It provides easy access to a globally distributed cache and scalable connection pool for serverless applications via a powerful API. The goal of Accelerate is to improve response times and reduce database load. It works by caching data at the edge using established caching patterns that you control.
+
+While Accelerate is beneficial for all types of applications, being at the edge provides additional benefits to edge function environments like [Vercel Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions), [Cloudflare Workers](https://workers.cloudflare.com/), and [Deno Deploy](https://deno.com/deploy). Cache hits can be served from data centers near the user regardless of the region of the database.
+
+<Admonition type="info">
+
+Accelerate is in Preview. While we have high confidence in our product, the nature of a preview product is that significant iterations might happen at any time, so we would advise against placing it in a system that requires stability.
+
+</Admonition>
+
+<Admonition type="info">
+
+All database queries not served from the cache (eg: write queries, cache misses, etc.) are routed through a [database connection pool](https://www.prisma.io/dataguide/database-tools/connection-pooling#what-is-connection-pooling). Currently, this infrastructure is only available in two regions: West Virginia and Ireland. In case the database is hosted far away from the Accelerate infrastructure, latency for queries to the database might be negatively affected. We are working on adding support for more regions.
+
+</Admonition>
+
+</TopBlock>
+
+## See Accelerate in action
+
+We built a small sample application, [Accelerate Speed Test](https://accelerate-speed-test.prisma.io/). The app compares the performance of cached and uncached queries side by side. The app is [open source](https://github.com/prisma/accelerate-speed-test) and you can clone it to try it yourself.
+
+<img
+  src="../../doc-images/data-platform/platform/accelerate/speed-test.png"
```

Comment:
@nikolasburk the speed test will be live with the preview launch.

Reply:
what's the reason we're waiting? is it not done yet or are we intentionally holding it back?

Reply:
@nikolasburk we're intentionally waiting for the launch

Reply:
@nikolasburk Does this mean I should leave the screenshot as is for now?

Reply:
Let's use a screenshot of the new one since by the time we merge the docs, the speed test will be merged as well: 

<img width="1072" alt="image" src="https://github.com/prisma/docs/assets/4058327/445b5057-bbf5-43a8-a38e-304fd0b2eccc">


- PR: https://github.com/prisma/accelerate-speed-test/pull/8
- Preview: https://accelerate-speed-test-git-ankur-minor-tweaks-prisma-adp.vercel.app/#testArea

Reply:
(maybe take a new screenshot in incognito mode)